.github/workflows/CICD.yml

```
@@ -150,7 +150,7 @@ jobs:
       uses: actions-rs/cargo@v1
       with:
         command: test
-        args: --features "feat_os_unix"
+        args: --features "feat_os_unix" -p uucore -p coreutils
```

please remove this
```suggestion
        args: --features "feat_os_unix"
```

please create a separate PR for this with the rational

---

created #2030 which now blocks this pr :)